  

# **Promote the use of biomarkers of aging in trials- through an open international collaboration methodology**

We are asking for contributions from the international community to inform the development of a short slide deck to stimulate discussion and engagement with key stakeholders on the use of aging biomarkers in trials in a collaborative process with a view to achieving international consensus on standards, validation and open decentralized data sharing.


# Contributing to the Presentation

We welcome contributions to our presentation! If you would like to contribute, please follow the steps outlined below.

## How to Contribute

1. **Fork the Project**: Start by forking the project repository on GitHub. You can do this by visiting [https://github.com/nikhilYadala/aging_biomarkers](https://github.com/nikhilYadala/aging_biomarkers) and clicking on the 'Fork' button.

2. **Download the Presentation**: After forking, download the presentation file. You can find the latest version in the PDF or PPTx format in the master branch. Alternatively, you can check the details of the latest merged commit, which includes the URL of the Google slides.

3. **Edit the Presentation**: Make your edits to the presentation. We prefer if you use Google slides for editing. This ensures uniformity and easier collaboration.

4. **Commit Your Edits**: Once you are done with your edits, commit the changes to your forked repository. Please use the following format for your commit message: `<<your description here>> URL: <<Google slides URL>>`. For an example of a properly formatted commit, refer to [this commit](https://github.com/nikhilYadala/aging_biomarkers/commit/b893d3c97ac97bf454cc8dbc24d0f9a3446900b4).

5. **Send a Pull Request**: Finally, send a pull request to the master branch of the main repository at [https://github.com/nikhilYadala/aging_biomarkers](https://github.com/nikhilYadala/aging_biomarkers). One of our core team volunteers will review your submission and, if appropriate, merge it into the main presentation. For an example of a properly formatted pull request, refer to [this PR](https://github.com/nikhilYadala/aging_biomarkers/pull/1).

## License

This presentation is available under the Creative Commons license. You are free to edit, distribute, and fork this presentation for your own purposes under the terms of this license.

Thank you for your interest in contributing to our project!



## About our efforts

**_Rationale_**

Biomarkers could accelerate and shorten clinical trial times by acting as surrogate endpoints upstream in the health trajectory, and measure risk and progression of major age-related diseases.  However, there Is no agreed standard or list of approved aging biomarkers for clinical trials; and there is no international consensus or set of validated biomarkers of aging or aging clocks.  There is also a need to move beyond static to more dynamic ways to measure resilience in response to stressor events and capture the complex and multifactorial processes underlying aging in human systems. This gap can be addressed by data-intensive omics, capture of digital biomarkers through ‘effortless AI’ (eg wearables) and application of machine learning, quantum computing and systems biology.

**_Target audience_**

Government and policymakers, regulators, investors, industry, entrepreneurs, scientists, academic researchers

**_Key Headings/topics in the presentation_**

**Why?** 

- Explore the role of biomarkers and clocks as endpoints for clinical trials (trustworthy, verifiable, transparent results) to accelerate and focus research

- Promote the use of biomarkers of aging to inform studies of baseline characteristics of participants in future and past clinical trials

- Use biomarkers along with chronological age when it comes to the table of baseline characteristics listed in trials

- Balance control and interventions/treatments based on biomarkers

- Lay the groundwork for future surrogate endpoints of interventions to delay age-related diseases.

- Build foundational genomic data base that allows researchers to assess causality due to the intervention

- Create a virtuous cycle: leverage biomarkers to find interventions that affect biological age, and can measure adaptation of human system to exposomic stresses (‘dynamic resilience’). Conversely: leverage successful interventions to develop better biomarkers.

- Leverage recent innovations and provide clinical decision support for FDA (and MHRA in UK)- that move beyond monocausal framework

- Harness opportunities in UK with government manifesto commitment to increase healthy life expectancy and minimise health & wellbeing inequalities:

  - biology of ageing is one of 7 core health challenges in [Life Science Vision ](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1013597/life-sciences-vision-2021.pdf)- and looking at aging biomarkers in upstream prevention trials

  -  pro-innovation data and digital reforms opening up access to consumer data, sharing public and private sector data, and accelerating digital clinical trials 

* Leverage opportunities to access UK Biobank data, [Nightigale Health Blood Sample Bank](https://nightingalehealth.com/news/nightingale-healths-strategic-collaboration-with-uk-biobank-progresses-biomarker-results-of-additional-200-000-blood-samples-soon-available-to-researchers-globally/) and inform protocols of [Our Future Health ](https://ourfuturehealth.org.uk/)sample collection to incorporate wider exposomic factors and interventions linked to healthspan (eg diet, exercise, socio-economic) and resilience (ability to adapt to stress)

**Evidence review of aging biomarkers to date**

- Not just for drugs but other types of interventions that are being studied for impact on healthspan and resilience (eg intermittent fasting, response to stress)

 

**Infrastructure for trials, open data sharing platform**

- Include details about the existing infrastructure for decentralized trials, open source

- The challenge lies in acquiring, extracting, transforming, and standardizing unstandardized data export formats and structures and integrating them into a standardized structure that allows for seamless analysis of aging biomarkers.

- Another challenge involves quantifying the effectiveness of longevity interventions, revealing hidden factors exacerbating aging, and determining personalized optimal daily values for these factors.

- An ideal approach is to enable users to publish their findings and decrease errors in predictive analysis by expanding the user sample size through data grouping from relatively homogeneous user groups.

- Leveraging machine learning techniques, the platform could provide predictive insights. These could potentially highlight early signs of age-related health changes before they become clinically apparent, thus contributing to the early detection and prevention of age-related diseases.

- The data-sharing feature of the platform could also facilitate collaborative research. Multiple researchers from various backgrounds and regions could collaborate on shared data, potentially speeding up the discovery of aging biomarkers.

- The platform could enable secure data sharing, permitting researchers and healthcare professionals to access this data. The aggregated data could then be used in broader studies aiming to identify biomarkers of aging.

- The platform could compile and analyze data over long periods, an essential feature for studying aging, which is inherently a longitudinal process.

- In summary, an open-source digital health platform could revolutionize our understanding of aging by providing large-scale, real-world, longitudinal data on health parameters and their changes with age. Nevertheless, it's important to ensure strict data security and privacy measures to protect the sensitive personal health information of users.

**The flow and the processes**

- Create an agreed list of biomarkers and aging clock endpoints  

- Protocol for anyone running a clinical trial collects a blood tube for measuring proteomics, methylation, and metabolomics (including continuously monitored digital parameters from wearables): it is relatively easy and cheap to collect blood and to generate methylation/proteomics data

-

**What should be our next steps?** 

- Including biomarkers that not only distinguish between chronological age and biological age but also change within a relatively short period after starting preventative interventions (such as diet and exercise) and therapeutics. 

* Moving from static to dynamic biomarkers to measure the adaptation of the human system to exposome stresses (‘dynamic resilience). What new set of biomarkers/ methods of identifying these biomarkers should be considered?

* Including organ-specific aging biomarkers and clocks as endpoints (because we can look for increased risk of morbidities, which is easier to measure than all-cause-mortality) - How do we better account for the organ-specific aging, and how do these aging clocks blend with the full body aging while running clinical trails.

  - Aging trials so far have a history of using age-related diseases like Alzheimer’s as the endpoint.

  - It’s much easier to convince FDA and other government organizations to focus on organ-specific age-related deterioration than all-cause mortality

  - We already have organ specific risk assessment clocks for Brain, Liver, and Kidney, and immune system (immune-Age - it’s not an organ per se, but a sub process of aging)

**Need to create clocks, metrics with easily screenable biomarkers**

- For building bias-less  Machine learning and AI models, it is essential that the data be representative of all the demographics that the aging clocks & interventions would be applicable. Towards this end, there is a need to build more practical metrics (that can be used in trials) using continuously monitored wearable information and standard blood biochemistry panels that are available all over the world for anyone to screen cheaply

- Our efforts to build more accurate aging clocks using other modalities of information can help us create approximate surrogate clocks from the regularly cheap-to-screen wearables and blood biochemistry markers.

**Consolidation of existing research work**

- Determinants of accelerated metabolomic and epigenetic aging in a UK cohort\
  Markers of biological aging have potential utility in primary care and public health. We developed a model of age based on untargeted metabolic profiling across multiple platforms, including nuclear magnetic resonance spectroscopy and liquid chromatography-mass spectrometry in urine and serum, within a large sample (N = 2,239) from the UK Airwave cohort. We validated a subset of model predictors in a Finnish cohort including repeat measurements from 2,144 individuals. We investigated the determinants of accelerated aging, including lifestyle and psychological risk factors for premature mortality. The metabolomic age model was well correlated with chronological age (mean r = .86 across independent test sets). Increased metabolomic age acceleration (mAA) was associated after false discovery rate (FDR) correction with overweight/obesity, diabetes, heavy alcohol use and depression. DNA methylation age acceleration measures were uncorrelated with mAA. Increased DNA methylation phenotypic age acceleration (N = 1,110) was associated after FDR correction with heavy alcohol use, hypertension and low income. In conclusion, metabolomics is a promising approach for the assessment of biological age and appears complementary to established epigenetic clocks.

&#x20;<https://pubmed.ncbi.nlm.nih.gov/32363781/> 

- Unsupervised learning of aging principles from longitudinal data -\
  Age is the leading risk factor for prevalent diseases and death. However, the relation between age-related physiological changes and lifespan is poorly understood. We combined analytical and machine learning tools to describe the aging process in large sets of longitudinal measurements. Assuming that aging results from a dynamic instability of the organism state, we designed a deep artificial neural network, including auto-encoder and auto-regression (AR) components. The AR model tied the dynamics of physiological state with the stochastic evolution of a single variable, the “dynamic frailty indicator” (dFI). In a subset of blood tests from the Mouse Phenome Database, dFI increased exponentially and predicted the remaining lifespan. The observation of the limiting dFI was consistent with the late-life mortality deceleration. dFI changed along with hallmarks of aging, including frailty index, molecular markers of inflammation, senescent cell accumulation, and responded to life-shortening (high-fat diet) and life-extending (rapamycin) treatments.\
  <https://www.nature.com/articles/s41467-022-34051-9> 

- Heterogeneous aging across multiple organ systems and prediction of chronic disease and mortality -\
  Biological aging of human organ systems reflects the interplay of age, chronic disease, lifestyle and genetic risk. Using longitudinal brain imaging and physiological phenotypes from the UK Biobank, we establish normative models of biological age for three brain and seven body systems. Here we find that an organ’s biological age selectively influences the aging of other organ systems, revealing a multiorgan aging network. We report organ age profiles for 16 chronic diseases, where advanced biological aging extends from the organ of primary disease to multiple systems. Advanced body age associates with several lifestyle and environmental factors, leukocyte telomere lengths and mortality risk, and predicts survival time (area under the curve of 0.77) and premature death (area under the curve of 0.86). Our work reveals the multisystem nature of human aging in health and chronic disease. It may enable early identification of individuals at increased risk of aging-related morbidity and inform new strategies to potentially limit organ-specific aging in such individuals.\
  <https://www.nature.com/articles/s41591-023-02296-6> 

- The Hallmarks of Aging\
  Aging is characterized by a progressive loss of physiological integrity, leading to impaired function and increased vulnerability to death. This deterioration is the primary risk factor for major human pathologies including cancer, diabetes, cardiovascular disorders, and neurodegenerative diseases. Aging research has experienced an unprecedented advance over recent years, particularly with the discovery that the rate of aging is controlled, at least to some extent, by genetic pathways and biochemical processes conserved in evolution. This review enumerates nine tentative hallmarks that represent common denominators of aging in different organisms, with special emphasis on mammalian aging. These hallmarks are: genomic instability, telomere attrition, epigenetic alterations, loss of proteostasis, deregulated nutrient-sensing, mitochondrial dysfunction, cellular senescence, stem cell exhaustion, and altered intercellular communication. A major challenge is to dissect the interconnectedness between the candidate hallmarks and their relative contribution to aging, with the final goal of identifying pharmaceutical targets to improve human health during aging with minimal side-effects.

<https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836174/>

- Biological Age Estimation Using Circulating Blood Biomarkers\
  Biological Age (BA) captures physiological deterioration better than chronological age and is amenable to interventions. Blood-based biomarkers have been identified as suitable candidates for BA estimation. This study aims to improve BA estimation using machine learning models and a feature-set of circulating biomarkers available from the UK Biobank (UKBB) (n = 307,000). We implement an Elastic-Net derived Cox model with 25 selected biomarkers to predict mortality risk, which outperforms the well-known blood-biomarker based PhenoAge model, providing a 9.2% relative increase in predictive value. Importantly, we then show that using common clinical assay panels, with few biomarkers, alongside imputation and the model derived on the full set of biomarkers, does not substantially degrade predictive accuracy from the theoretical maximum achievable for the available biomarkers. BA is estimated as the equivalent age within the same-sex population which corresponds to an individual’s mortality risk. Values ranged between 20-years younger and 20-years older than individuals’ chronological age, exposing the magnitude of ageing signals contained in blood markers. Thus, we demonstrate a practical and cost-efficient method of estimating an improved measure of BA, available to the general population

<https://www.medrxiv.org/content/10.1101/2023.02.23.23285864v1.full.pdf/> 

-  Longitudinal analysis of blood markers reveals progressive loss of resilience and predicts human lifespan limit

We investigated the dynamic properties of the organism state fluctuations along individual aging trajectories in a large longitudinal database of CBC measurements from a consumer diagnostics laboratory. To simplify the analysis, we used a log-linear mortality estimate from the CBC variables as a single quantitative measure of the aging process, henceforth referred to as dynamic organism state indicator (DOSI). We observed, that the age-dependent population DOSI distribution broadening could be explained by a progressive loss of physiological resilience measured by the DOSI auto-correlation time. Extrapolation of this trend suggested that DOSI recovery time and variance would simultaneously diverge at a critical point of 120 − 150 years of age corresponding to a complete loss of resilience. The observation was immediately confirmed by the independent analysis of correlation properties of intraday physical activity levels fluctuations collected by wearable devices. We conclude that the criticality resulting in the end of life is an intrinsic biological property of an organism that is independent of stress factors and signifies a fundamental or absolute limit of human lifespan

<https://www.nature.com/articles/s41467-021-23014-1> 


**Action plan for stakeholders** to reach an international global consensus on guidelines for measurement

- Bring the above-proposed infrastructure for decentralized trials and an open data-sharing platform to life. Most of it is already built at [www.curedao.org](http://www.curedao.org) 

- Use the existing D2C apps to curate and normalize the continuous longitudinal data collected from wearables, blood work, and all other health information to allow people to opt-in for decentralized clinical trials ( This will be a proof-of-concept for the FDA to see how effective such digital platforms could be in terms of recruiting, accessing the data, and analyzing the data to determine statistical outcomes. We need to show an example to the governments all over the world that 

  - Our metrics to measure aging mean a literal decrease in the risk of disease and death

  - Our methodology of decentralized trials means that one need not spend a billion dollars for each small trials

  - We can do “personalized” trials successfully and establish (backed by experiment) a model among variables such as intermittent fasting, water intake, and supplements to end-metrics such as sleep scores, energy scores, and aging clocks. 

  - Most of the necessary regulatory hurdles for anonymized data collection, privacy-preserving federated learning, and zero-knowledge proofs for authenticating users without spilling personally identifiable information have already been built by our friends at <https://www.weavechain.com/>  and <https://flemingprotocol.io/>

  - Some examples of such centralised experiments are <https://www.nature.com/articles/s41746-022-00630-9-> Age estimation from sleep studies using deep learning predicts life expectancy, <https://www.nature.com/articles/s41598-022-24053-4>  Association between vitamin D supplementation and COVID-19 infection and mortality



**End Goal**

Get FDA to approve aging clocks/biomarkers  as surrogate endpoints, and test various aging drugs and lifestyle interventions to create models that can inform personalized health optimization protocols.
